nim/rosencrantz/server.nimble

```diff
@@ -0,0 +1,11 @@
+# Package
+
+version = "0.1.0"
+author = "Willyboar"
+description = "Rosencrantz Implementation"
+license = "MIT"
+
+# Dependencies
+
+requires "nim >= 1.0.0"
+requires "rosencrantz >= 0.4.3"
```

Comment:
Could you use a range, `>= 0.4.3 & < 0.5`

Reply:
Done!